### DISCUSSION TITLE
🚀 Use ZigReleaser (GoReleaser) for building & releasing Ghostty

### DISCUSSION THREAD

**Discussion by vedantmgoyal9** [ghostty-org/ghostty#3050] [Feature Requests, Ideas] [answered]
Title says it all. GoReleaser now supports building and releasing Zig projects.

As we approach Ghostty's public release, I’m unsure if @mitchellh would want to modify the existing release CI at this stage. If it’s fine, I can open a PR to add the GoReleaser configuration and update the CI. Otherwise, we could consider this for the future.

/cc @caarlos0

**Comment by caarlos0** [comment]
FWIW I can help setting things up!

**Comment by mitchellh** [comment, marked as answer]
I'm a big fan of GoReleaser and used it heavily in my Go days. I definitely don't want to rock the boat when our release process is already working great, but I'm open to this _if it leads to real improvements_. I don't want to change something for change sake, so I think more research would have to be done on how it can improve our current process.

Bear in mind a couple of the most critical aspects of the Ghostty project:

1. We don't maintain official builds or packages _except_ macOS binaries in dmg form since signing/notarization costs money.

2. We must enable industry-standard package managers to package Ghostty in a process approved way. Example: https://github.com/ghostty-org/ghostty/blob/main/PACKAGING.md
